docs/content-types/react-content-types.md

```diff
@@ -0,0 +1,213 @@
+---
+sidebar_label: For React
+sidebar_position: 1.5
+---
+
```

Comment:
Added this as its own standalone doc for now because the configuration for React is different from the handling described in the individual content type sections that already exist.

Let's see if we can rearchitect this doc to fit into the existing pattern - but not required for release. I think we need to refactor the granularity of docs as we work on moving the README content into the .org docs. This can be a part of that.

Reply:
makes sense to be standalone for now